sktime/forecasting/compose/_hierarchy_ensemble.py

```diff
@@ -0,0 +1,431 @@
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements forecaster for applying different univariates on hierarchical data."""
+
+__author__ = ["VyomkeshVyas", "ciaran-g"]
```

Comment:
did @ciaran-g contribute to this?

Reply:
Hi, I've just been trying to help @VyomkeshVyas when he has Qs or is stuck - I haven't actually contributed any code so far

Reply:
then it wouldn't be authorship but more of an acknowledgment (if desired)

Reply:
thanks for helping btw!

Reply:
> then it wouldn't be authorship but more of an acknowledgment (if desired)

I agree tbh, @VyomkeshVyas if you could remove me from here it would be appreciated 👍

Reply:
> These are genuine failures now.
> 
> Have you run `check_estimator` locally and tried to debug?
> 
> I would recommend that, see here: https://www.sktime.org/en/stable/developer_guide/add_estimators.html

Apologies, I didn't check that previously. I am working on it now.